core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -134,7 +134,7 @@ private SqlLibraryOperators() {
 
   /** The "DATE_ADD(date, numDays)" function
    * (Spark) Returns the date that is num_days after start_date. */
-  @LibraryOperator(libraries = {SPARK})
+  @LibraryOperator(libraries = {SPARK, ORACLE, HIVE})
```

Comment:
Does hive support both `DATE_ADD(date, interval)` and `DATE_ADD(date, numDays)`?

Reply:
Thanks for reminder, hive not support both,  only support DATE_ADD(date, numDays), I would change it. @ILuffZhe